source/styleguide/components/SgColorSwatch/SgColorSwatch.jsx

```diff
@@ -0,0 +1,154 @@
+import SgHeading from '@sg-tags/SgHeading/SgHeading';
+import { createObject, getContrast, runWCAGTest } from './SgColorSwatch.Container';
+
+/* variables file generated by the postcss-export-vars plugin. */
+import colorVars from './SgColorSwatch__variables.json';
+
+export const SgColorSwatch = (props) => {
+  const {
+    tagName: Tag,
+    className,
+    children,
+    ...attrs
+  } = props;
+
+  const classStack = FcUtils.createClassStack([
+    'SgColorSwatch',
+    className
+  ]);
+
+  return (
+    <SgColorSwatch__wrapper>
+      {
+        colorVars ?
+        Object.keys(colorVars).map((color, i) => {
+          const title = color.substr(color.indexOf('color') + 5).split(/(?=[A-Z])/).join(' ');
+          const obj = createObject(colorVars[color]);
+
+          /* contrast tests */
+          const contrastPrimary = getContrast(obj.hex, colorVars.colorTextPrimary);
+          const contrastSecondary = getContrast(obj.hex, colorVars.colorTextSecondary);
+
+          return (
+            <Tag
+              key={i}
+              className={classStack}
+              {...attrs}
+              style={{ backgroundColor: obj.hex }}
+            >
+              <SgColorSwatch__accessibility variant="no-badge" contrastPrimary={contrastPrimary} contrastSecondary={contrastSecondary} level="AA" />
```

Comment:
I vote to keep the no-badge variant off by default. It took me a while to find the context of the PASS/FAIL tests with out seeing the actual content of each badge unfortunately.

Reply:
This was an accident! Putting the proper fix in :)